src/components/Column.ts

```diff
@@ -111,7 +113,7 @@ export default (editor: grapesjs.Editor, { opt, coreMjmlModel, coreMjmlView, san
       },
 
       getChildrenSelector() {
-        return 'table';
+        return 'table > tbody > tr > td > table';
```

Comment:
This seems to break the Column component

Reply:
Yes in order to inner background to work i need adjust the getChildrenSelector()
and i had check even with this change the background color will work as normal.
If you need i can provide the mjml code you need to compare
Thanks

Reply:
I'm just trying to load the default template and this is what I see
<img width="965" alt="Screenshot 2023-06-21 at 11 31 19" src="https://github.com/GrapesJS/mjml/assets/11614725/bdf6dd67-9697-4ced-a262-98a72b549fb2">

that means the output returned from mjml compiler doesn't match your new selector